src/components/dashboard/subscriptions/Subscribe.js

```diff
@@ -1,16 +1,23 @@
 import React, {Component, PropTypes} from 'react';
-import {Link} from 'react-router';
+import {browserHistory} from 'react-router';
+
+import FaArrowLeft from 'react-icons/lib/fa/arrow-left';
 
 class Subscribe extends Component {
     render() {
         const inputClass = 'input-reset ba b--silver pa3 mv2 db br3 mh3';
-        const btnClass = 'pointer dim br1 ba bw1 ph4 pv3 black mr3 b';
+        const btnClass = 'pointer dim br1 ba bw1 ph2 pv3 black b';
         const linkClass = 'no-underline black';
         const rowClass = 'mv3';
 
         console.log(this.props.bean);
         return (
             <div className="content">
+                <div style={{width: 100 + 'px'}}>
+                    <div className={btnClass} onClick={browserHistory.goBack}>
```

Comment:
More logic is necessary to handle that case.  We shouldn't display a back button, then, imo...  not sure yet.  Very rare that a user would get in that situation I think.